tool/make-native-distribution.sh

```diff
@@ -123,7 +123,10 @@ fi
 "$release_home/bin/ruby" -v
 
 cd "$build_home"
+# All specs except C-extensions
 tool/jt.rb test --native -t "$release_home/bin/truffleruby"
+# C-extensions specs
+tool/jt.rb test --native :capi :library_cext -t "$release_home/bin/truffleruby"
```

Comment:
I'm surprised combining `--native` with `-t` works.

Reply:
Ah right, I should use AOT_BIN?

Reply:
With `-t` it uses a mix of the build image and the release image, so using AOT_BIN seems better.

Reply:
With AOT_BIN it looks fine, except the home of `jt` is used, which ideally it would not be and the home could be the one of the release binary.

Reply:
d1e0f84393e637af0a4659c16e54456393999af2

Reply:
You could set `TRUFFLERUBYOPT=-Xhome...`. Or `Utilities.find_launcher` in jt.rb could be modified to avoid the mess of environment variables. Another option is avoiding `--native` and just copying over the list of exclusions here (`graalvm` and `aot`).